engine/src/main/java/org/terasology/rendering/dag/nodes/DownSamplerNode.java

```diff
@@ -113,9 +110,6 @@ public void process() {
 
     @Override
     public void update() {
-        // Note: we don't need to update the inputFBO because only SetInputTextureFromFBO
-        // needs to keep it up to date and takes care of that internally.
-        outputFBO = outputFBOmanager.get(outputFBOconfig.getName());
+        outputFbo = outputFboManager.get(outputFboUrn);
```

Comment:
That comment would have made sense, had there been an inputFbo variable.

(I removed it, because it was useless)